great_expectations/zep/logger.py

```diff
@@ -0,0 +1,30 @@
+import logging
+import sys
+
+
+def init_logger(level: int = logging.WARNING):
+    print("setting up logger ...")
+    try:
+        from loguru import logger
```

Comment:
@billdirks would you like me to leave this in here or remove it?

I was going to remove it on the PR and just modify it locally, but if you also find this useful I can leave it in.

Reply:
Will move this to the demo repo.